tutorials/content_filtering/cs/README.md

```diff
@@ -0,0 +1,46 @@
+# Tutorial: Content Filtering
+
+This code is part of the Connext
+[Content Filtering](https://community.rti.com/static/documentation/developers/learn/content-filtering.html)
+tutorial.
+
+## Building the Example :wrench:
+
+You can build the example following the instructions in the tutorial, or you can
+build it using CMake as follows.
+
+1.  Generate the C# types and the project files file with **rtiddsgen**:
+
+```sh
+<install dir>/bin/rtiddsgen -language C# -platform [net5|net6|net8] home_automation.idl
+```
+
+Where `<install dir>` refers to your RTI Connext installation.
+
+2.  Build the applications:
+
+```sh
+dotnet build
+```
+
+## Running the Example :rocket:
+
+Running publisher
```

Comment:
```suggestion
Run the publisher:
```

Would you mind updating the READMEs for the other C# examples from "Running X" to "Run the X:" as well?

Reply:
I updated the other c# related READMEs. I noticed that the optional things are marked: Extra. Run the ...

Would you like me to change that as well to: (Optional) Run the ...